include/openspace/scene/scene.h

```diff
@@ -203,6 +207,17 @@ class Scene
     std::mutex _programUpdateLock;
     std::set<ghoul::opengl::ProgramObject*> _programsToUpdate;
     std::vector<std::unique_ptr<ghoul::opengl::ProgramObject>> _programs;
+
+    struct InterpolationInfo {
+        // @TODO(abock): The property might have been deleted underneath us, which would
```

Comment:
Good point: I think this needs to be fixed before it can be merged.
would it be good/feasable to change the interface of property owners to store shared pointers instead of raw ones? We would need to change pretty much all concrete property owners...